velox/docs/functions/presto/math.rst

```diff
@@ -27,6 +27,15 @@ Mathematical Functions
     verified for performance reasons. Returns ``high`` for all values of ``x``
     when ``low`` is greater than ``high``.
 
+.. function:: cosine_similarity(map(varchar, double), map(varchar, double)) -> double
+
+    Returns the cosine similarity between the sparse vectors x and y. If any value of any key is NULL,
```

Comment:
perhaps, turn "cosine similarity" into a link to https://en.wikipedia.org/wiki/Cosine_similarity to provide convenient access to unfamiliar readers